cmd/boots/tftp.go

```diff
@@ -16,10 +18,15 @@ import (
 	"github.com/tinkerbell/boots/job"
 	"github.com/tinkerbell/boots/metrics"
 	tftp "github.com/tinkerbell/tftp-go"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
 )
 
 var (
-	tftpAddr = conf.TFTPBind
+	tftpAddr      = conf.TFTPBind
+	traceparentRe = regexp.MustCompile("^(.*)-[[:xdigit:]]{2}-([[:xdigit:]]{32})-([[:xdigit:]]{16})-([[:xdigit:]]{2})")
```

Comment:
We could make it like `${filename}-otel-${traceparent}` and I agree that would be easier to spot and tell what's going on. I didn't hammer on this too much because I wasn't sure of the value or danger level of it. If it seems ok in general I'm happy to tighten it up.

Reply:
yep, doesn't raise any neck hairs over here, :+1:

Reply:
could this possibly be moved down into `func extractTraceparentFromFilename` where it's being used? this is the same pattern you used for `pxeVendorOptions`. And is actually in line with Boots' design philosophy (see 5 and 6 in the `General` section [here](https://github.com/tinkerbell/boots/blob/main/docs/DESIGNPHILOSOPHY.md#general))

Reply:
moved in dfed6517f82432c4ae038cb756dcbeadc5f8bdcd